test/Dapr.Client.Test/ConfigurationApiTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Autogenerated = Dapr.Client.Autogen.Grpc.v1;
+using Xunit;
+using FluentAssertions;
+
+namespace Dapr.Client.Test
+{
+    public class ConfigurationApiTest
+    {
+        [Fact]
+        public async Task GetConfigurationAsync_ValidateRequest()
+        {
+            await using var client = TestClient.CreateForDaprClient();
+            var metadata = new Dictionary<string, string>
+            {
+                { "key1", "value1" },
+                { "key2", "value2" }
+            };
+            var request = await client.CaptureGrpcRequestAsync(async daprClient =>
+            {
+                return await daprClient.GetConfiguration("testStore", new List<string>() { "test_key" }, metadata);
+            });
+
+            request.Dismiss();
```

Comment:
Can you change this to return some data instead of doing nothing? Example: https://github.com/halspang/dotnet-sdk/blob/ae9f30e59cbd02f383f29c07af4ef41d96659941/test/Dapr.Client.Test/DaprClientTest.InvokeMethodGrpcAsync.cs#L123

Then we can validate the output as well.